### PR TITLE
add pymssql to win/req.txt

### PR DESCRIPTION
**Pull request by cmcmarrow**
### What does this PR do?
adds pymssql to win req.txt

### What issues does this PR fix or reference?
https://github.com/saltstack/salt/issues/55484

### Tests written?
Yes

### Commits signed with GPG?
Yes

### CODE REVIEW

**Review by xeacott — APPROVED** [review]

File: .pre-commit-config.yaml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 exclude: ^(doc/_static/.*|doc/_themes/.*)$
 repos:
   - repo: https://github.com/saltstack/pip-tools-compile-impersonate
-    rev: master
+    rev: 999c70e967636c579e919bc2bb30985b4c9e996d
```
Comment (Akm0d) [inline .pre-commit-config.yaml]: This should stay as `master`

File: .pre-commit-config.yaml [hunk 2]
```diff
@@ -390,7 +390,7 @@ repos:
 
 
   - repo: https://github.com/saltstack/salt-nox-pre-commit
-    rev: master
+    rev: 52423a28436dbb49e1c401ed11c3836bdf21bd44
```
Comment (Akm0d) [inline .pre-commit-config.yaml]: and this

File: .pre-commit-config.yaml [hunk 3]
```diff
@@ -402,7 +402,7 @@ repos:
           - --
 
   - repo: https://github.com/saltstack/salt-nox-pre-commit
-    rev: master
+    rev: 52423a28436dbb49e1c401ed11c3836bdf21bd44
```
Comment (Akm0d) [inline .pre-commit-config.yaml]: and this

File: .pre-commit-config.yaml (hunk 1 above)
Comment (s0undt3ch) [reply .pre-commit-config.yaml]: Yes, it should. Why did you need to change it?

**Review by s0undt3ch — DISMISSED** [review]
What @Akm0d said.

**Review by s0undt3ch — COMMENTED** [review]
Erm... There's a bug in our pip tools compile stuff.
I'm setting a reminder to update your PR and push the changes tomorrow morning.

**Review by waynew — APPROVED** [review]

**Comment by s0undt3ch** [comment]
Done!

**Comment by cmcmarrow** [comment]
Thanks

**Review by s0undt3ch — APPROVED** [review]

**Comment by cmcmarrow** [comment]
rerun-full centos6

**Comment by cmcmarrow** [comment]
re-run full centos6

**Pull request merged**